plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/AssignAnalysis.java

```diff
@@ -355,6 +355,21 @@ private TokensList getNonFunctionDefCompletionsFromAssign(ICodeCompletionASTMana
                                     }
                                 }
                             }
+                            if (args instanceof Str) {
+                                Str str = (Str) args;
+                                List<String> attribs = StringUtils.split(str.s, ',');
```

Comment:
I think you could actually simplify it a bit by doing: 
```
for (String s : str.s.split("(\\s|,)+")) {
    if(!s.isEmpty()) {
        ....
    }
}
```

Also, `str.s` needs to be checked for `null`.